scripts/build-version-info.mjs

```diff
@@ -0,0 +1,31 @@
+import { readFileSync, writeFileSync } from 'node:fs'
+import { execSync } from 'node:child_process'
+import path from 'node:path'
+import url from 'node:url'
+
+const dirname = path.dirname(url.fileURLToPath(import.meta.url))
+const root = path.join(dirname, '..')
+
+const pkg = JSON.parse(readFileSync(path.join(root, 'package.json'), 'utf8'))
+
+function tryCommand(cmd) {
+    try {
+        return execSync(cmd, { stdio: ['ignore', 'pipe', 'ignore'] }).toString().trim()
+    } catch {
+        return ''
```

Comment:
Shouldn't we at least log error if the command fails?

Reply:
Thanks! I've [updated](https://github.com/fingerprintjs/dx-team-mock-for-proxy-integrations-e2e-tests/pull/47/commits/2d92c3a3317061bb1e93631721f94068224cc0f0) the script.